packages/block-library/src/latest-posts/edit.js

```diff
@@ -411,6 +446,12 @@ class LatestPostsEdit extends Component {
 										__( '(no title)' )
 									) }
 								</a>
+								{ displayAuthor && (
+									<div className="wp-block-latest-posts__post-author">
+										{ __( 'Posted by' ) }{ ' ' }
```

Comment:
👋 This should be incorporating the author's name like in #20595 

```JS
<span className="wp-block-latest-posts__post-author">
	{ sprintf(
		/* translators: byline. %s: current author.
		__( 'by %s' ),
		currentAuthor.name
	) }
</span>
```

Reply:
I agree this is a better approach. I've updated my code accordingly.